t/stream/xrpc/downstream.t

```diff
@@ -56,14 +56,14 @@ timeout
 
 
 === TEST 3: read with peek
+--- ONLY
 --- stream_server_config
     preread_by_lua_block {
         local ffi = require("ffi")
-        local sk = require("resty.apisix.stream.xrpc.socket").downstream.socket()
+        local sk = ngx.req.socket(true)
```

Comment:
```lua
require("resty.apisix.stream.xrpc.socket").downstream.socket()
```
is equivalent to

```lua
ngx.req.socket(true)
```

minus the read, read_line and move methods that are not being used currently.

this is done in order to repro the bug with vanilla openresty.